Common/SecurityIdentifier.cs

```diff
@@ -38,6 +38,7 @@ public struct SecurityIdentifier : IEquatable<SecurityIdentifier>
         #region Empty, DefaultDate Fields
 
         private static readonly string MapFileProviderTypeName = Config.Get("map-file-provider", "LocalDiskMapFileProvider");
+        private static readonly char[] InvalidCharacters = {'|', ' '};
```

Comment:
Thank you. This should have been added when we originally implemented/designed the serialization/parsing scheme